fitlins/interfaces/nistats.py

```diff
@@ -122,18 +122,33 @@ def _run_interface(self, runtime):
         smoothing_fwhm = self.inputs.smoothing_fwhm
         if not isdefined(smoothing_fwhm):
             smoothing_fwhm = None
+
         flm = level1.FirstLevelModel(
+            minimize_memory=False,
```

Comment:
I guess it's this line that's killing us?

Reply:
Yes and no. 

Yes because that argument must be set to true for those images to be saved to that object. Otherwise, because they are not super necessary they are deleted along with a bunch of other stuff.

@jdkent maybe there's another way to access these images without triggering a bunch of other stuff to be kept around?

Reply:
From what I can tell, that argument saves the `SimpleRegressionResult` object, and without it, you can't get the other model outputs. 

nistats is also archived, so proposing an alternative for now is not possible.

Maybe we can just make this optional in fitlins? I'll raise an issue over there and see what they say.